API/things-api/app.js

```diff
@@ -206,8 +238,10 @@ app.get('/stats/range/:start_date/:end_date', function(err, client, done) {
 
 /****************************************************
 * /path     /stats/range/:start_date/:end_date
```

Comment:
This path name does not go with this function.  Should be something like  FUNC_NAME.